### PR TITLE
CC-14610 Unzer integration guide

### PR DESCRIPTION
**Pull request by kkicheglovspryker**
## PR Description

Jira Ticket: https://spryker.atlassian.net/browse/CC-15101

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/user/technology-partners/202204.0/payment-partners/unzer/project-integration-unzer.md [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+last_updated: Jun 07, 2022
+template: concept-topic-template
+originalLink: https://documentation.spryker.com/202204/docs/unzer-integration
+originalArticleId:
+related:
+- title: Installing and configuring Unzer
+  link: docs/scos/dev/technology-partner-guides/page.version/payment-partners/unzer/install-and-configuration-unzer.html
+---
+
+This article provides step-by-step instructions on integrating the Adyen module into your project.
```
Comment (michbeck) [inline docs/scos/user/technology-partners/202204.0/payment-partners/unzer/project-integration-unzer.md:13]:
```suggestion
This article provides step-by-step instructions on integrating the Unzer module into your project.
```

File: docs/scos/user/technology-partners/202204.0/payment-partners/unzer/install-and-configuration-unzer.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## Installation
+
+To install the Unzer modules, run the command:
+```
+composer require spryker-eco/unzer spryker-eco/unzer-gui
+```
+
+## General Configuration
+
```
Comment (michbeck) [inline docs/scos/user/technology-partners/202204.0/payment-partners/unzer/install-and-configuration-unzer.md:41]: Would recommend to split this part into two subpoints

### Unzer configuration
### Unzer API configuration

As well shouldn't we list all possible configuration keys? Seems those from API are missing.

**Review by michbeck — CHANGES_REQUESTED** [review]
I know this [Feature integration guides](https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides) does not meet 100% the eco requirements, but as far as i know we should add verification and some others for our modules as well.

**Comment by michbeck** [comment]
Should the file docs/scos/user/technology-partners/202200.0/payment-partners/unzer/unzer.md really be empty? Shouldn't there be Partner information e.g. like this:

`[Unzer](https://www.unzer.com/) is one of the leading payment companies in Europe. Over 70,000 merchants trust in our end-to-end solutions for more growth – online, mobile, and at the point of sale. Merchants can choose from data-driven services, including international payment processing, risk management, and customer behaviour analysis. This means our merchants only need one partner for future-proof, flexible, and innovative payment solutions.

Unzer was founded in 2020 as a merger of multiple specialised fintech companies. Uniting these companies under one brand enables Unzer to make unified commerce a reality.`

File: docs/scos/user/technology-partners/202204.0/payment-partners/unzer/install-and-configuration-unzer.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    $config[UnzerConstants::WEBHOOK_RETRIEVE_URL] = 'https://yves.spryker.com/unzer/notification';
+
+```
+
+## Expense refund strategies
+You can choose one of provided refund strategies for your payment flow:
+* `UnzerConstants::LAST_SHIPMENT_ITEM_EXPENSES_REFUND_STRATEGY` - Expense costs will be refunded with the last item in corresponding shipment.
+* `UnzerConstants::LAST_ORDER_ITEM_EXPENSES_REFUND_STRATEGY` - Expense costs will be refunded with the last item in whole order.
+* `UnzerConstants::NO_EXPENSES_REFUND_STRATEGY` - Expense costs will not be refunded.
+
```
[the remark this answers is not in the document]
Comment (kkicheglovspryker) [reply docs/scos/user/technology-partners/202204.0/payment-partners/unzer/install-and-configuration-unzer.md:105]: Key is provided in config table

**Review by michbeck — APPROVED** [review]

**Review by michbeck — APPROVED** [review]

File: docs/scos/user/technology-partners/202204.0/payment-partners/unzer/install-and-configuration-unzer.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+- title: Unzer integration guide
+  link: docs/scos/dev/technology-partner-guides/page.version/payment-partners/unzer/project-integration-unzer.html
+---
+
+This topic describes how to install and configure the Unzer module to integrate Unzer into your project.
+
+## Integration overview
+
+The `spryker-eco/unzer` module provides integration of Spryker e-commerce system with Unzer technology partner. It requires the `SprykerEco.UnzerApi` `spryker-eco/unzer-api` module that provides the REST Client for making API calls to the Unzer Payment Provider.
+It also requires `SprykerEco.UnzerGui` `spryker-eco/unzer-gui` module that provides Backoffice functionality to configure Unzer credentials.
```
Comment (DmitryLymarenko) [inline docs/scos/user/technology-partners/202204.0/payment-partners/unzer/install-and-configuration-unzer.md:18]: why do we duplicate module naming here using different approaches?

> `SprykerEco.UnzerGui` `spryker-eco/unzer-gui`
> `SprykerEco.UnzerApi` `spryker-eco/unzer-api`

Comment (jeremyfourna) [reply docs/scos/user/technology-partners/202204.0/payment-partners/unzer/install-and-configuration-unzer.md:18]: What do you mean @DmitryLymarenko ?

**Review by DmitryLymarenko — APPROVED** [review]

**Review by jeremyfourna — APPROVED** [review]

**Pull request closed without merging**